app/src/main/java/com/besome/sketch/editor/property/PropertyInputItem.java

```diff
@@ -170,15 +170,18 @@ private void showViewIdDialog() {
         aB dialog = new aB((Activity) getContext());
         dialog.b(tvName.getText().toString());
         dialog.a(icon);
-        View view = wB.a(getContext(), R.layout.property_popup_input_text);
-        EditText input = view.findViewById(R.id.ed_input);
-        input.setLines(1);
-        _B validator = new _B(context, view.findViewById(R.id.ti_input), uq.b, uq.a(), jC.a(sc_id).a(projectFileBean), value);
+
+        PropertyPopupInputTextBinding binding = PropertyPopupInputTextBinding.inflate(LayoutInflater.from(getContext()));
+
+        binding.tiInput.setHint(String.format(Helper.getResString(R.string.property_enter_value), "widget ID"));
+
+        binding.edInput.setLines(1);
+        _B validator = new _B(context, binding.tiInput.findViewById(R.id.ti_input), uq.b, uq.a(), jC.a(sc_id).a(projectFileBean), value);
```

Comment:
why not just use 
```
binding.tiInput
```

Reply:
Yes, that's right.
I must have missed it